dd-java-agent/instrumentation/play-2.8/play-2.8.gradle

```diff
@@ -0,0 +1,55 @@
+// Modified by SignalFx
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+  // Play doesn't work with Java 9+ until 2.6.12
+  maxJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+def scalaVersion = '2.13'
+def playVersion = '2.7.3'
+
+muzzle {
+  pass {
+    group = 'com.typesafe.play'
+    module = "play_$scalaVersion"
+    versions = "[$playVersion,)"
+    assertInverse = true
+  }
+  pass {
+    group = 'com.typesafe.play'
+    module = 'play_2.12'
+    versions = '[2.6.8, )'
+    assertInverse = true
+  }
+}
+
+apply from: "${rootDir}/gradle/java.gradle"
+
+apply plugin: 'org.unbroken-dome.test-sets'
+
+testSets {
+  latestDepTest {
+    dirName = 'test'
+  }
+}
+
+dependencies {
+  main_java8Compile group: 'com.typesafe.play', name: "play_$scalaVersion", version: playVersion
+
+  testCompile project(':dd-java-agent:instrumentation:java-concurrent')
+  testCompile project(':dd-java-agent:instrumentation:netty-4.0')
+  testCompile project(':dd-java-agent:instrumentation:netty-4.1')
+  testCompile project(':dd-java-agent:instrumentation:akka-http-10.0')
```

Comment:
If you add the play 2.4 and 2.6 projects are there any conflicts or issues?

Reply:
Issues: All the tests for the 2.8 async server fail.

Reply:
That is likely a sign this instrumentation, as is, introduces breaking behavior.